types/node/test/buffer.ts

```diff
@@ -368,6 +368,14 @@ b.fill("a").fill("b");
     b = a.readBigUint64BE(123);
 }
 
+{
+    const buf = Buffer.allocUnsafe(5);
+    buf.fill("a");
+    buf.fill("aazz", "hex");
+    buf.fill("aazz", 1, "hex");
+    buf.fill("aazz", 1, 2, "hex");
```

Comment:
Nit: should test the return type.

```suggestion
    const buf = Buffer.allocUnsafe(5);
    let result: Buffer;
    result = buf.fill("a");
    result = buf.fill("aazz", "hex");
    result = buf.fill("aazz", 1, "hex");
    result = buf.fill("aazz", 1, 2, "hex");
```

Reply:
I updated in 0c4247e.